### ISSUE TITLE
Questions about  experimental results

### ISSUE THREAD

**Issue by zw615** [krumo/Detectron-DA-Faster-RCNN#7]
Hi, I have some questions regarding the performance of your code.
The original paper's performance ( using the SIM 10k dataset as the source domain and the Cityscapes training set as the target domain) is much lower than your code.

original:
![图片](https://user-images.githubusercontent.com/26880977/58847384-5680e080-86b5-11e9-9be5-625763fb300c.png)

yours:
![图片](https://user-images.githubusercontent.com/26880977/58847488-be372b80-86b5-11e9-82d7-d95066c0d632.png)

I wonder why. Did you add some additional tricks or the implementation of original paper is buggy? Also, can your result be reproduced stably?
Wonderful work, by th way. Thanks a lot

**Comment by krumo** [comment]
Hi, as I mentioned in the README, 

> The best results for different adaptation are reported. Due to the instable nature of adversarial training, the best models are obtained through a model selection on a randomly picked mini validation set.

The original implementation and original paper don't mention these model selection things. Although there exists some difference between my implementation and the original implementation, I believe model selection is the key reason why my results seem higher. This repo only aims to reproduce the results in the original paper. It works but the performance is quite unstable. The PyTorch version I'm developing seems more stable. I didn't run the original code in Caffe and it might be able to yield a more stable performance. 

So in my opinion, the instability might also comes from difference between deep learning framework. It's an interesting topic, but unfortunately I'm current occupied by a few other projects and don't have time and computing resources to explore it. Welcome to pull request for any enhancement!
